scripts/st2bootstrap-deb.template.sh

```diff
@@ -104,6 +104,34 @@ setup_args() {
         exit 1
     fi
   fi
+
+  # Python 3.6 package is not available in Ubuntu Xenial
+  # Installer can add it via 3rd party PPA based on user agreement
+  if [[ "$SUBTYPE" = 'xenial' ]]; then
+    sudo apt-get update > /dev/null 2>/dev/null
+    # check if python3.6 is available
+    if (! apt-cache show python3.6 2> /dev/null | grep 'Package:' > /dev/null); then
+      echo ""
+      echo "WARNING!"
+      echo "python3.6 package is required as a dependency for StackStorm and none of your software repositories provide it."
+      echo "We recommend switching to Ubuntu 18.04 LTS (Bionic) as a base OS."
+      echo ""
+      echo "Alternatively we'll try to add 3rd party python3.6 APT repository: https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa."
+      echo "By continuing you're aware of security risk associated with using unofficial 3rd party PPA repository."
```

Comment:
```suggestion
      echo "The python3.6 package is a required dependency for the StackStorm st2 package but that is not installable from any of the default Ubuntu 16.04 repositories."
      echo "We recommend switching to Ubuntu 18.04 LTS (Bionic) as a base OS. Support for Ubuntu 16.04 will be removed with StackStorm 3.5."
      echo ""
      echo "We will use python3.6 from the 3rd party 'deadsnakes' PPA: https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa."
      echo ""
      echo "By continuing you are aware of the support and security risks associated with using unofficial 3rd party PPA repository, and you understand that StackStorm does NOT provide ANY support for python3.6 packages on Ubuntu 16.04."
      echo ""
      echo "To bypass this check in the future and automatically install and use the deadsnakes PPA, you can run this script with:"
      echo ""
      echo "echo 'y' | $(0) $@"
```

Reply:
Thanks, I took some of the fixes you suggested: https://github.com/StackStorm/st2-packages/pull/681/commits/aaeea508253b5c41314f932635928988ecb25d86

I won't include a command example with `y` and I'd prefer to avoid a wall of text to keep the information amount somewhat balanced. I liked more explicit wording around security risks, - that's very helpful and more clear :+1: